manifests/database.pp

```diff
@@ -1,5 +1,8 @@
-# Set up the foreman database
-class foreman::database {
+# @summary Set up the foreman database
+# @api private
+class foreman::database(
+  Integer $timeout = 0,
```

Comment:
You can set a lower bound (and upper bound, but I don't think that makes sense here)
```suggestion
  Integer[0] $timeout = 0,
```

Reply:
done